File: src/Sketch.java
import processing.core.PApplet;

public class Sketch extends PApplet {

  static final int COL = 800;
  static final int ROW = 600;
  static final int ITE = 100;
  static final float ZOOM = 0.00375f;

  public static void main(String[] args) {
    PApplet.main(new String[]{"Sketch"});
  }

  public void settings() {
    size(COL, ROW);
  }

  public void setup() {
    background(255);
    float a, b, ac, bc, aa, bb;
    for (int Y = 0; Y < ROW; Y++) {
      for (int X = 0; X < COL; X++) {
        a = ac = map(X, 0, COL-1, -COL*ZOOM/3*2, COL*ZOOM/3);
        b = bc = map(Y, 0, ROW-1, -ROW*ZOOM/2, ROW*ZOOM/2);

        int n = 0;
        while (n < ITE) {
          aa = a*a - b*b;
          bb = 2*a*b;
          a = aa + ac;
          b = bb + bc;
          if (abs(a+b) > 1600000) break;
          n++;
        }
        set(X, Y, color(map(n, 0, ITE, 0, 255)));
      }
    }
  }

  public void draw() {

  }
}

File: sketch.js
'use strict';

const ite = 100
const zom = 0.00375

function setup() {
  createCenteredCanvas()
  noLoop()
}

function windowResized() {
  createCenteredCanvas()
  draw()
}

function createCenteredCanvas() {
  // const size =
  //   (windowWidth <= 540 || windowHeight <= 630) +
  //   (windowWidth <= 960 || windowHeight <= 800)
  // const w = [800, 600, 400][size]
  // const h = [600, 450, 300][size]
  createCanvas(800, 600).position(
    (windowWidth - width) / 2,
    (windowHeight - height) / 2
  )
}

function draw() {
  background(255, 0, 200)
  for (let y = 0; y < height; y++) {
    for (let x = 0; x < width; x++) {
      let a = map(x, 0, width - 1, -width * zom / 3 * 2, width * zom / 3)
      let b = map(y, 0, height - 1, -height * zom / 2, height * zom / 2)
      let ac = a
      let bc = b
      let n = 0
      while (n < ite && (abs(a + b) < 1600000)) {
        let aa = a * a - b * b
        let bb = 2 * a * b
        a = aa + ac
        b = bb + bc
        n++
      }
      set(x, y, color(map(n, 0, ite, 0, 255)))
    }
  }
  updatePixels()
}
